### PR TITLE
No more dynamic ayys

### PR DESCRIPTION
**Pull request by ghost**
They're annoying, unfun, and repetitive.

:cl:
del: Abductors are much less common now.
/:cl:

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
so we let monkeys run around but not abductors?
why not just lower the rate they spawn in? that seems to be the main complaint

File: hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -64,46 +64,6 @@
...
 	return TRUE
-
-
-/datum/dynamic_ruleset/roundstart/abductors
-	name = "Abductors"
-	antag_flag = ROLE_ABDUCTOR
-	protected_roles = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
-	restricted_roles = list("Cyborg")
-	required_candidates = 2
-	weight = 30
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:75]: you could instead just lower this

**Comment by TheTrigg** [comment]
I agree with dwarf. They spawn A LOT. Pretty much every time dynamic mode has points to spare. I guess the threat points should be increased and weight should be reduced so abductor spawns stop occuring in almost every round.

Almost everyone hates abductors over here at beat!.
I think the main reason for the hate isn't even the sleep from the baton (except for when the scientist doesn't know how to play and the agent has to keep resleeping them until they figure it out), I think it's the glands. Most of them have an ANNOYING AS FUCK effect. Random brain traumas, plasma farts, shitting out slimes, depending on the gland you got the round can be pretty much over for someone. Which is pretty much why I asked for stuff like #12029 .

**Comment by diegoxt45** [comment]
> I agree with dwarf. They spawn A LOT. Pretty much every time dynamic mode has points to spare. I guess the threat points should be increased and weight should be reduced so abductor spawns stop occuring in almost every round.
> 
> Almost everyone hates abductors over here at beat!.
> I think the main reason for the hate isn't even the sleep from the baton (except for when the scientist doesn't know how to play and the agent has to keep resleeping them until they figure it out), I think it's the glands. Most of them have an ANNOYING AS FUCK effect. Random brain traumas, plasma farts, shitting out slimes, depending on the gland you got the round can be pretty much over for someone. Which is pretty much why I asked for stuff like #12029 .

git gud

**Comment by ghost** [comment]
They also spawn alot here.

Don't like em either.

**Comment by WarRage333** [comment]
They are more of an annoyance then a real threat, so just put em as midround and we gucci

**Comment by TheTrigg** [comment]
> just put em as midround and we gucci

May I suggest the opposite? Having abductors spawn only on roundstart and never on midround?

The problem with spawning on midround is that... Well, time has passed. People are more geared, and chances are half the station is already dead. And abductors don't greentext off of dead bodies.
So spawning them midround makes their objectives harder due to a lack of.. People being alive.

Whereas spawning them on roundstart gives them time and subjects to abduct, and also time for the station to prepare against them while hell hasn't completely broken loose yet. Also fixes #12034 in a pinch.

**Comment by TheTrigg** [comment]
Optionally also make them leave once they've reached their target amount of points so they don't get to be a worry and/or start sleeping people just for fun for the entire round.

**Comment by ghost** [comment]
@TriggeredBoi then they'll stop _just short_ of their objective and continue boning

**Comment by ghost** [comment]
@WarRage333 is right -- they aren't really a big enough threat to be roundstart

**Comment by Majick1** [comment]
@steamp0rt if that's a problem then just make the goal a hidden value but have it be something 4 and under so it doesn't drag on too long.

**Comment by cris5311** [comment]
-1

reduce spawnrate and/or increase threat but dont just remove something because you are too lazy to balance it.

**Comment by ghost** [comment]
they'll still happen midround

**Comment by ghost** [comment]
they're just not threatening nor fun enough for a roundstart threat

**Comment by karmaisblackandbluepilled** [comment]
-1 just lower the rate or always have abductors also have 1 traitor spawned along them

**Comment by AnthonyJNgo** [comment]
+1 tbh, abductors are lame imo. Unfun and the instateleport is extremely gimmicky.

**Comment by yoyobatty** [comment]
I don't usually weigh my opions here but it would be better if you simply reduced the spawn rate.

**Comment by Toxici11i** [comment]
abductors just need a reduced spawn rate.
Also they occur so much due to them being near-perfect for lowpop, but can roll an abduct objective higher than the total crew count.
Keep ayys, reduce spawnrate or increase threat

**Comment by Maroonenahmoron** [comment]
Lower the chance of roundstart abductors and if possible have them spawn along side other antagonists. Mid-round abductors can go wrong if the station is already killed off so that’s better off being disabled.

**Comment by ghost** [comment]
@Maroonenahmoron there is no magic "make them only spawn with other antags" value in dynamic

**Comment by Maroonenahmoron** [comment]
@steamp0rt Well, lowering the chance of abductors will be enough then.

**Comment by ghost** [comment]
fine i'll just raise the minimum pop

**Comment by ghost** [comment]
yeah i just halfed the weight

**Comment by Maroonenahmoron** [comment]
Pretty sure that'll do, +1

**Comment by cris5311** [comment]
you did what i asked for, so here is the +1 i promised.

**Review by ghost — APPROVED** [review]

**Pull request merged**